package.json

```diff
@@ -70,7 +70,8 @@
     "test:tools": "jest ./tools/ -c ./tools/jest.config.js",
     "test:tools-16": "USE_REACT_16=true jest ./tools/ -c ./tools/jest.config.js",
     "test:coverage": "yarn pre-test && jest --coverage",
-    "test:website": "start-server-and-test 'yarn serve:website' http://localhost:9000 'yarn cypress run --record --spec ./cypress/integration/**/*.spec.ts'",
+    "test:website": "start-server-and-test 'yarn serve:website' http://localhost:9000 'yarn cypress run --record --parallel --group cypress-tests --spec \"./cypress/integration/**/**/*.spec.ts\"'",
```

Comment:
ℹ️ Parallelized tests in cypress are available in a CI environment when:
- the run command has `--parallel=true`
- the run command has `--record=true`
- the CI environment config specifies the number of machines to run
Also note that we added a grouping label to this command.
It could be helpful in the future to split out our tests into separate jobs and group them by type/category.

🪙 🪙 The biggest value add here is that it could help us analyze our testing data in the future via the cypress dashboard & api.